backend_fen.go

```diff
@@ -160,3 +160,9 @@ func (w *Watcher) Add(name string) error {
 func (w *Watcher) Remove(name string) error {
 	return nil
 }
+
+// SetRecursive enables watches to also monitor subdirectories. Currently
+// only supported under Windows.
+func (w *Watcher) SetRecursive() error {
+	return nil
```

Comment:
[Question]
If it's only supported on Windows, why this one does not retunr an error?

Reply:
Not the most satisfying answer, but because that is what the original code does.  It was in `fen.go`

https://github.com/adriansr/fsnotify/blob/a81f2b630e7c708d59a259e6308cd70409a1ebd3/fen.go#L39-L43